rma/models/res_users.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    rma_team_id = fields.Many2one(
+        'crm.team', "User's RMA Team",
```

Comment:
Should be this, isn't it?
```suggestion
        'rma.team', "User's RMA Team",
```

Reply:
yes, changed